M/Mineos/build_tarballs.jl

```diff
@@ -3,19 +3,37 @@
 using BinaryBuilder
 
 name = "Mineos"
-version = v"1.0"
+version = v"1.0.2" 
```

Comment:
Where is v1.0.2 coming from?  Latest tag [I can see](https://github.com/geodynamics/mineos/tags) is v1.0.1

Reply:
1.0.2 is present in the commit message, the version number is also provided in a file, but not an explicit tag